src/app/app.module.ts

```diff
@@ -7,22 +7,33 @@ import { HeroesComponent } from './heroes/heroes.component';
 import { HeroDetailComponent } from './hero-detail/hero-detail.component';
 import { MessagesComponent } from './messages/messages.component';
 import { DashboardComponent } from './dashboard/dashboard.component';
+import { HttpClientModule } from '@angular/common/http';
+import { HttpClientInMemoryWebApiModule } from 'angular-in-memory-web-api';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/infinity-arc/Kobus-tour-of-heroes/pullRequest?prid=7838883)
```suggestion
import { HttpClientInMemoryWebApiModule } from "angular-in-memory-web-api";
```